code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,330 @@
+/*TODO:
+	PTSD
+*/
+/datum/brain_trauma/psychological
+	var/cure_points = 1000
+	var/current_points = 0
+	var/fearless_mod = 1
+	var/lithiated_mod = 1
+	var/relaxed_mod = 1
+	var/control_mod = 1
+
+/datum/brain_trauma/psychological/on_life()
+	if(HAS_TRAIT(owner, TRAIT_FEARLESS))
+		current_points += 2 * fearless_mod
+	if(HAS_TRAIT(owner, TRAIT_LITHIATED))
+		current_points += 15 * lithiated_mod
+	if(HAS_TRAIT(owner, TRAIT_RELAXED))
+		current_points += 5 * relaxed_mod
+	if(HAS_TRAIT(owner, TRAIT_UNDER_CONTROL))
+		current_points += 10 * control_mod
+	if(current_points >= cure_points)
+		SEND_SIGNAL(src, COMSIG_CURE_MENTAL, src)
+
+
+/datum/brain_trauma/psychological/depression
+	name = "Depression"
+	desc = "Extensive damage to receptors in brain result in subject's neigh constant bad mood"
+	scan_desc = "Electric imaging suggests a mental disorder is present"
```

Comment:
should these have periods? check the other brain traumas and see if their scan_desc values end with periods